honeybee_energy/measure.py

```diff
@@ -0,0 +1,428 @@
+# coding=utf-8
+"""Module for parsing OpenStudio measures and setting measure arguments."""
+from __future__ import division
+
+import os
+import xml.etree.ElementTree as ElementTree
+
+
+class Measure(object):
+    """Object to hold all properties of an OpenStudio measure, including arguments.
+
+    Args:
+        folder: Path to the folder in which the measure exists. This folder
+            must contain a measure.rb and a measure.xml file. Other files are
+            optional.
+
+    Properties:
+        * folder
+        * xml_path
+        * rb_path
+        * identifier
+        * display_name
+        * description
+        * type
+        * arguments
+    """
+
```

Comment:
It is not required by `__slots__` are missing from this class.

Reply:
I was debating whether I should implement it since we are probably going to be adding more to the class in the future. I'll add it for now.